extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -93,6 +107,7 @@ class AmpLightbox extends AMP.BaseElement {
     if (this.active_) {
       return;
     }
+    this.isScrollable_ = this.element.hasAttribute('scrollable');
```

Comment:
@camelburrito suggested moving it to `buildCallback` and I think it's the right place